src/components/Cart/CartScreen.js

```diff
@@ -175,54 +222,84 @@ function CartScreen() {
         <Typography className={classes.cartHeader}>
           Cart
         </Typography>
-        <Typography className={classes.farmLabel}>
-          Shopping from Farm
-        </Typography>
+        {!loading && (cartListings.length !== 0 ? (
```

Comment:
This ternary implies that the empty cart screen will be shown if the loading state is false and if there are no listings. I thought this would prevent the empty cart screen from being shown until the listings are loaded and could be checked, but I think loading changes before the cartListings are fully loaded and I'm not sure how to fix that.

Reply:
No worries about that at all right now! It's not a huge issue, and I have a feeling it has to do with our structure of airtable queries, but in all honesty if there's a bit of loading that the client sees, I'm totally okay with that